forge-gui/res/cardsfolder/upcoming/xanders_wake.txt

```diff
@@ -0,0 +1,7 @@
+Name:Xander's Wake
+ManaCost:1 B
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigDraft | ActivationLimit$ 1 | TriggerDescription$ Whenever one or more creatures you control die, draft a card from CARDNAME's spellbook. This ability triggers only once each turn.
+SVar:TrigDraft:DB$ Draft | Spellbook$ Acquisitions Expert,Asylum Visitor,Blade Juggler,Bloodthirsty Aerialist,Boneclad Necromancer,Hoard Robber,Malakir Blood-Priest,Malakir Cullblade,Morbid Opportunist,Slaughter Specialist,Tavern Swindler,Thieves' Guild Enforcer,Tithebearer Giant,Vengeful Warchief,Yuan-Ti Fang-Blade
+DeckHas:Ability$Sacrifice
```

Comment:
DeckHints not Has because this card doesn't actually enable sacrificing... unless a card in the spellbook does...

Reply:
Should probably update DeckHas to include pointers for the stuff in the Spellbook